flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/RemoteStreamEnvironment.java

```diff
@@ -204,48 +203,9 @@ public RemoteStreamEnvironment(String host, int port, Configuration clientConfig
 		this.savepointRestoreSettings = savepointRestoreSettings;
 	}
 
-	/**
-	 * Executes the job remotely.
-	 *
-	 * <p>This method can be used independent of the {@link StreamExecutionEnvironment} type.
-	 * @return The result of the job execution, containing elapsed time and accumulators.
-	 */
-	@PublicEvolving
-	public static JobExecutionResult executeRemotely(StreamExecutionEnvironment streamExecutionEnvironment,
```

Comment:
@kl0u I'll approve the pull request after you check [the comment](https://github.com/apache/flink/pull/7249#issuecomment-554171248) with @tweise . Maybe this method not in used is still valid.

Reply:
I think this can be removed. The same can now be accomplished by using `Executor`/`ExecutorFactory` directly.